electroncash_plugins/fusion/plugin.py

```diff
@@ -653,6 +655,84 @@ def donation_address(self, window) -> Optional[Tuple[str,Address]]:
         if self.remote_donation_address and Address.is_valid(self.remote_donation_address):
             return (self.fullname() + " " + _("Server") + ": " + self.get_server()[0], Address.from_string(self.remote_donation_address))
 
+    @staticmethod
+    def wallet_can_fuse(wallet) -> bool:
+        return can_fuse_from(wallet) and can_fuse_to(wallet)
+
+    @classmethod
```

Comment:
Nit: I think this could be a `@staticmethod`

Reply:
True. I will add a commit to address this nit.